### PR TITLE
Add stock string 173 (Member added)

### LINKED ISSUE #2834: Add stock string 173 (Member added)
It was added in https://github.com/deltachat/deltachat-core-rust/pull/4952

### PR DESCRIPTION
**Pull request by adbenitez**
close #2834

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -802,6 +802,8 @@
     <!-- %1$s will be replaced by name and address of the contact who did the action -->
     <string name="group_image_changed_by_other">Group image changed by %1$s.</string>
     <!-- %1$s will be replaced by name and address of the contact added to the group -->
+    <string name="member_added">Member %1$s added.</string>
```
Comment (r10s) [inline res/values/strings.xml:805]: to not confuse with the plain text "member added" (even if not in used currently), maybe change the key to:
```suggestion
    <string name="member_x_added">Member %1$s added.</string>
```

File: src/org/thoughtcrime/securesms/connect/DcHelper.java [hunk 2]
```diff
@@ -255,6 +255,7 @@ public static void setStockTranslations(Context context) {
     dcContext.setStockTranslation(171, context.getString(R.string.chat_protection_broken_tap_to_learn_more));
 
     dcContext.setStockTranslation(172, context.getString(R.string.chat_new_group_hint));
+    dcContext.setStockTranslation(173, context.getString(R.string.member_added));
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/connect/DcHelper.java:258]: same here:
```suggestion
    dcContext.setStockTranslation(173, context.getString(R.string.member_x_added));
```

**Review by r10s — APPROVED** [review]
slight rewording of key suggested, otherwise lgtm

**Comment by adbenitez** [comment]
> slight rewording of key suggested, otherwise lgtm

thanks! :+1:

**Pull request merged**